packages/react/src/experimental/OneDataCollection/visualizations/collection/Card/index.tsx

```diff
@@ -220,7 +226,7 @@ const GroupCards = <
         const primaryAction =
           itemActions.find((action) => action.type === "primary") || undefined
         const secondaryActions =
-          itemActions.filter((action) => action.type === "secondary") || []
+          itemActions.filter((action) => action.type === "secondary") 
```

Comment:
I'd rather kept the invariant that `secondaryActions` is always an array and checked it's size in the place we render